todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsViewModel.java

```diff
@@ -17,38 +17,84 @@
 package com.example.android.architecture.blueprints.todoapp.statistics;
 
 import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+import android.databinding.ObservableBoolean;
+import android.support.annotation.VisibleForTesting;
 
+import com.example.android.architecture.blueprints.todoapp.BR;
 import com.example.android.architecture.blueprints.todoapp.R;
 import com.example.android.architecture.blueprints.todoapp.data.Task;
+import com.example.android.architecture.blueprints.todoapp.data.source.TasksDataSource;
+import com.example.android.architecture.blueprints.todoapp.data.source.TasksRepository;
+import com.example.android.architecture.blueprints.todoapp.util.EspressoIdlingResource;
 
 import java.util.List;
 
 /**
- * Exposes the data to be used in the {@link StatisticsContract.View}.
+ * Exposes the data to be used in the {@link StatisticsFragment}.
  * <p>
  * Note that in this case the view model is also the view, not the fragment.
```

Comment:
The ViewModel is also the View? 
Shouldn't classes have only one responsibility? If you want to keep it like this it would be nice to add one more line explaining why do we have to make this a View.